examples/readme_test.rs

```diff
@@ -74,9 +94,10 @@ impl From<embedded_sdmmc::SdCardError> for Error {
 }
 
 fn main() -> Result<(), Error> {
-    let sdmmc_spi = FakeSpi();
-    let sdmmc_cs = FakeCs();
+    let spi_bus = RefCell::new(FakeSpiBus());
```

Comment:
Please change the README to match :)

Reply:
The stuff you highlighted are the fake devices that will be replaced by real hardware in actual use cases.
The remainder of test is the same as in the Readme.md